build.gradle

```diff
@@ -43,59 +46,57 @@ buildscript {
         }
         mavenCentral()
     }
-    ext {
-        projectVersion = '1.3.0-RELEASE'
-        grpcVersion = '1.60.0'
-        protobufVersion = '3.21.12'
-        protobufGradlePluginVersion = '0.8.0'
-    }
 
     dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:2.0.2'
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3'
-        classpath "gradle.plugin.com.liferay:gradle-plugins-node:4.3.0"
+        classpath 'com.github.jengelman.gradle.plugins:shadow:5.2.0'
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.12'
```

Comment:
In line 51, this version is 7.1.2 in java-tron.

Reply:
which dependency ?

Reply:
classpath "gradle.plugin.com.github.johnrengelman:shadow:7.1.2" . This version of Shadow supports Gradle 7.0+ only. So 5.2.0 is available.

Reply:
@lxcmyf shadow is not necessary for java-tron. You can try to compile it without shadow.